internal/services/kaas/kaas_resource.go

```diff
@@ -260,6 +309,17 @@ func (r *kaasResource) Read(ctx context.Context, req resource.ReadRequest, resp
 		)
 	}
 
+	oidc, err := r.client.Kaas.GetOidc(kaasObject.Project.PublicCloudId, kaasObject.Project.ProjectId, kaasObject.Id)
+	if err != nil {
+		resp.Diagnostics.AddWarning(
+			"Could not get Oidc",
+			err.Error(),
+		)
+	}
+	state.OidcCaCertificate = types.StringValue(oidc.Certificate)
```

Comment:
issue: if `r.client.Kaas.GetOidc` returns an error, `oidc` is nil resulting in a panic while trying to access to `oidc.Certificate`